### PR TITLE
Add 5.8.0 post

### LINKED ISSUE #138: [Post]: 5.8.0 released
- Minetest no longer ships with a default game - instead, users are directed to choose one from the ContentDB dialog
   - There should a section explaining why this is important
- Redesigned the Settings GUI to be more user-friendly
- Android: Place nodes with single tap
- Added improved antialiasing filters
- Modding: Added ability to override item images using ItemMetaData

### PR DESCRIPTION
**Pull request by rubenwardy**
Fixes #138

### To do

- [x] Write text
- [x] Edit / polish text
- [x] Add cover image
- [x] Conclusion
- [x] ~~Add screenshot gallery (optional)~~

### CODE REVIEW

File: _posts/2023-12-04-5.8.0-released.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+loading long sounds when needed.
+
+Game developers and modders have more options on how to play sound, including
+changing the start time and offsets.
+
+
+## ...and more
+
+The antialiasing code has been improved and modernised. There's now the option
+to choose between two new algorithms - FXAA and MSAA. FXAA is faster but is
```
Comment (grorp) [inline _posts/2023-12-04-5.8.0-released.md:119]:
```suggestion
to choose between two new algorithms - FXAA and SSAA. FXAA is faster but is
```

Minetest already had MSAA before 5.8.0

File: _posts/2023-12-04-5.8.0-released.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+less accurate. MSAA is very accurate but is slower.
```
Comment (grorp) [inline _posts/2023-12-04-5.8.0-released.md:120]:
```suggestion
less accurate. SSAA is very accurate but is slower.
```

**Review by appgurueu — APPROVED** [review]
Gave it a read, looks fine. Ending seems a bit abrupt; you could consider putting something like a conclusion, or what to look forward to in future versions, there.

**Comment by TheShadowOfHassen** [comment]
I'd agree with @appgurueu, it needs an ending, a look forward would be nice. I'd personally like to read whether they'll be any improvements to minetest's built in "store"

**Comment by rollerozxa** [comment]
Should there be some sort of comparison image eyecandy for SSAA? I made this one a while back showing the difference between no antialiasing and x8 SSAA, looking at some terrain with the OhCeeDee texture pack enabled.

**Comment by TheShadowOfHassen** [comment]
> Should there be some sort of comparison image eyecandy for SSAA? I made this one a while back showing the difference between no antialiasing and x8 SSAA, looking at some terrain with the OhCeeDee texture pack enabled.

Perhaps. I thought that a "gallery" was only done during the monthly releases, but I think a picture would be nice.

**Comment by rubenwardy** [comment]
> Gave it a read, looks fine. Ending seems a bit abrupt; you could consider putting something like a conclusion, or what to look forward to in future versions, there.
> 
> I'd agree with @appgurueu, it needs an ending, a look forward would be nice. I'd personally like to read whether they'll be any improvements to minetest's built in "store"

Yeah I agree and thought the same, just wasn't sure what to write

File: _posts/2023-12-04-5.8.0-released.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: "Minetest 5.8.0 released!"
+author: rubenwardy
+description: >-
+  Minetest 5.8.0 has been released with improved onboarding, a new Settings
+  GUI, and improved Android controls
+image: /static/blog/5.8.0/cover.jpg
+image_credit: Wuzzy
+# forum_topic: https://forum.minetest.net/viewtopic.php?f=18&t=29356
```
Comment (srifqi) [inline _posts/2023-12-04-5.8.0-released.md:10]: The forum post now exists.
```suggestion
forum_topic: https://forum.minetest.net/viewtopic.php?t=30005
```

File: _posts/2023-12-04-5.8.0-released.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+changing the start time and offsets.
+
+
+## ...and more
+
+The anti-aliasing code has been improved and modernised. There's now the option
+to choose between two new algorithms - FXAA and SSAA. FXAA is faster but is
+less accurate. SSAA is very accurate but consumes more resources.
+
+{% include figure.html src="/static/blog/5.8.0/antialiasing.webp" caption="A magnified screenshot showing how anti-aliasing smoothes pixels (rollerozxa)" %}
```
Comment (srifqi) [inline _posts/2023-12-04-5.8.0-released.md:110]:
```suggestion
{% include figure.html src="/static/blog/5.8.0/antialiasing.webp" caption="A magnified screenshot showing how anti-aliasing smoothens pixels (rollerozxa)" %}
```

File: _posts/2023-12-04-5.8.0-released.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+{% include figure_gallery.html items=page.mtg_debundle %}
+
+
+## New Settings GUI
+
+Minetest 5.8.0 comes with a new settings menu, designed to make it easier for
+new users to find and change settings.
+
+Don't worry about the appearance - we're going to be giving the entire main
+menu a makeover in a future release.
```
Comment (rubenwardy) [inline _posts/2023-12-04-5.8.0-released.md:77]: Feels like this line could be improved. Perhaps something about it currently matching the current menu style, but the whole menu will be restyled in the future

File: _posts/2023-12-04-5.8.0-released.md (hunk 4 above)
Comment (rubenwardy) [reply _posts/2023-12-04-5.8.0-released.md:109]: "smoothens" doesn't sound correct

Comment (rubenwardy) [reply _posts/2023-12-04-5.8.0-released.md:109]: ahhh I meant smooths. Weird how that wasn't picked up by the spell checker

**Pull request merged**